nx/guides/modoc_agg_nx_01.livemd

```diff
@@ -0,0 +1,748 @@
+# Livebook companion for Nx aggregations
+
+```elixir
+Mix.install([
+  {:nx, "~> 0.4.2"}
+])
+```
+
+## Quick review on tensors with Nx
+
+```elixir
+import Nx, only: :sigils
+```
+
+A tensor can be viewed as a list of indexed numbers.
+
+#### A note on how to build tensors
+
+We will use the function `Nx.tensor/2` to [construct tensors](https://hexdocs.pm/nx/Nx.html#module-creating-tensors). A few examples:
+
+* you can pass a number to a tensor and you get a 0D tensor. For example, `Nx.tensor(1)` is a 0D tensor.
+
+```elixir
+Nx.tensor(1)
+```
+
+* you can build a 1D tensor or "vector" by passing a list of numbers. Equivalently, you can use the sigil `~V`. Both parenthesis "(" and brackets "[" are accepted.
+
+```elixir
+v = Nx.tensor([1, 2, 3])
+v1 = ~V(1 2 3)
+v2 = ~V[1 2 3]
+
+{
+  v == v1,
+  v == v2,
+  v
+}
+```
+
+* you can build a matrix or 2D tensor with the sigil `~M`. For example a matrix with [shape](<(https://hexdocs.pm/nx/Nx.html#shape/1)>) (2,3).
+
+```elixir
+m = ~M( 1 2 3
+  4 5 6)
+```
+
+Equivalently, you can write the structure by using nested lists. You can also start with a "flat vector" and _reshape_ it. For example:
+
+```elixir
+m1 = Nx.tensor([[1, 2, 3], [4, 5, 6]])
+
+m2 = ~V(1 2 3 4 5 6) |> Nx.reshape({2, 3})
+
+{
+  m == m1,
+  m == m2,
+  m
+}
+```
+
+* for higher dimensions, you can pass an arbitrarily nested list of numbers, or equivalently, you can reshape a "flat vector" into the dimensions that match your nested list. For example:
+
+```elixir
+t1 = Nx.tensor([[[1, 2, 3], [4, 5, 6]], [[7, 8, 9], [10, 11, 12]]])
+
+t2 = ~V(1 2 3 4 5 6 7 8 9 10 11 12) |> Nx.reshape({2, 2, 3})
+
+{
+  Nx.shape(t1),
+  t1 == t2,
+  t1
+}
+```
+
+#### A note on how to access to values of a tensor
```

Comment:
tying into my suggestion of splitting into 2 livebooks/guides, I think up until this section would be part of a more general "how to create and operate tensors" guide, and from the "Aggregations..." section onwards would stay in this guide.

Reply:
Yes..., useless here